### DISCUSSION TITLE
split argocd-notifications-cm in multiple files

### DISCUSSION THREAD

**Discussion by jgerits** [argoproj/argo-cd#18465] [Q&A]
We are gonna need a lot of triggers in argocd-notifications-cm is there an easy way to split this into multiple files so we dont get one really long file?

**Comment by agaudreault** [comment]
If you use kustomize, you can apply multiple patches and add different triggers to the same CM.
You can also remove the file from this deployment, and deploy the configmap independently in it's own applications, where you can manage it as a helm chart, kustomize or other supported tools of your choice.
